legacy/routes/lros.js

```diff
@@ -58,12 +62,30 @@ var lros = function (coverage) {
     res.status(201).type('json').end('{ "properties": { "provisioningState": "Succeeded"}, "id": "100", "name": "foo" }');
   });
 
-  coverage['LROPut200InlineCompleteNoState'] = 0;
   router.put('/put/200/succeeded/nostate', function (req, res, next) {
     coverage['LROPut200InlineCompleteNoState']++;
     res.status(200).type('json').end('{"id": "100", "name": "foo" }');
   });
 
+  router.patch("/patch/200/succeeded", function (req, res, next) {
+    coverage["LROPatchInlineComplete"]++;
+    var pollingUri = getRequestBaseUrl(req) + "/lro/patchasync/200/operationResults";
+    var headers = {
```

Comment:
To be consistent with the testserver naming convention, the test `LROPatchInlineComplete` of url `/patch/200/succeeded` should not have headers defined, like you can see the parallel in PUT:
https://github.com/Azure/autorest.testserver/blob/ab118a361d2b98697ed4b6ba9450348beeaf6c3d/legacy/routes/lros.js#L55-L58